README.dev.md

```diff
@@ -59,22 +62,40 @@ pre-commit run -a
 
 ## Contributing workflow
 
-- Always branch from `main`
+Our workflow is:
+
+1) Fetch from org remote, fast-forward your local main
+2) Create a branch to address the issue (see below for naming) - *"Always branch from `main`."*
+3) Push the new local branch to your personal remote repository
+4) Create a pull request to merge your remote branch into the org main
+
+Creating a branch:
 
-- Add a branch prefix
+- If there is an associated issue, add the issue number
+- If there is no associated issue, **and the changes are small**, add a prefix such as "typo", "hotfix", "small-refactor", according to the type of update
+- If the changes are not small and there is no associated issue, then create the issue first, so we can properly discuss the changes
 
-  - If there is an associated issue, add the issue number
-  - If there is no associated issue, **and the changes are small**, add a prefix such as "typo", "hotfix", "small-refactor", according to the type of update.
-  - If the changes are not small and there is no associated issue, then create the issue first, so we can properly discuss the changes
+Commit message:
 
-- Commit message
+- Use imperative, present tense (Add feature, Fix bug)
+- Have informative titles
+- If necessary, add a body with details
 
-  - Use imperative, present tense (Add feature, Fix bug)
-  - Have informative titles
-  - If necessary, add a body with details
+Before creating a pull request:
 
 - Try to create "atomic git commits" (recommended reading: [The Utopic Git History](https://blog.esciencecenter.nl/the-utopic-git-history-d44b81c09593))
 - Make sure the tests pass
 - Make sure the pre-commit tests pass
-- Before creating the pull-request, fetch any `main` updates from upstream and rebase your branch into `origin/main` if necessary
-- Open a pull request and work with the reviewer to address any issues
+- Rebase: Fetch any `main` updates from upstream and rebase your branch into `origin/main` if necessary
+- Then you can pen a pull request and work with the reviewer to address any issues
+
+## GIT Rules of Engagement
+
+- Assign only yourself to issues.
+- Assign yourself to issues you **want** to address. Consider if you will be able to work on it in the near future - if not, consider leaving it available for someone else to address.
+- Set the issue Status to "In Progress" when you have started working on it.
+  - Creating a PR for an issue (even if only a draft) will automatically set an issue as 'In Progress.' A good habit is creating a *draft* PR early, to take advantage of this automation and get feedback early.
+- When finalizing a PR, set the Status to "Ready for Review" and consider assigning a specific Reviewer.
```

Comment:
I think it's better to have Reviewers self-assign PRs and only request reviews when necessary.

Reply:
I changed the wording to: 
- When finalizing a PR, set the Status to "Ready for Review" - if someone specific **needs** to review it, you can assign them as the reviewer.